src/App.tsx

```diff
@@ -1,12 +1,63 @@
 import React from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { Goodslist } from './components/GoodsList/GoodsList';
+import * as goodsAPI from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[];
+};
+
+class App extends React.PureComponent<{}, State> {
+  state: State = {
+    goods: [],
+  };
+
+  displayGoods = async (selected : () => Promise<Good[]>) => {
+    const goods = await selected();
+
```

Comment:
What if error occurs?

Reply:
it wasn't in task, so first solution not include error cheker,
Now everything fixed.